README.md

```diff
@@ -14,6 +15,22 @@ Name | Description | Badges
 
 See [`ndk-examples`](./ndk-examples) for examples using the NDK and the README files of the crates for more details.
 
+## Supported NDK versions
+
+`android-ndk-rs` aims to support at least the `LTS` and `Rolling Release` branches of the NDK, as described on [their wiki](https://github.com/android/ndk/wiki#supported-downloads). Additionally the `Beta Release` might be supported to prepare for an upcoming release.
+
+As of writing (2021-07-24) the following NDKs are tested:
+
+Branch | Version | Status | Working
+-|-|:-:|:-:
+r18 | 18.1.5063045 | _Deprecated_ | :x:
+r19 | 19.2.5345600 | _Deprecated_ | :heavy_check_mark:
+r20 | 20.1.5948944 | _Deprecated_ | :heavy_check_mark:
+r21 | 21.4.7075529 | LTS | :heavy_check_mark:
+r22 | 22.1.7171670 | Rolling Release | :heavy_check_mark:
+r23 beta 1/2 |  | Beta | :heavy_check_mark:
+r23 beta 3 and beyond |  | Beta | :x: Breaking on #149 :x:
```

Comment:
the link to #149 doesn't work. curious, why doesn't r18 work? I guess we can remove the empty winit/glutin and flutter sections too.

Reply:
> the link to #149 doesn't work.

Ah that's a bummer, it's a hardlink now.

> curious, why doesn't r18 work? 

It doesn't have `<triple>-android<sdk>-clang(++)` binaries, just `clang`/`clang++`. I don't know if `clang` is supported in these NDKs at all or if that's just a host compiler and GCC is supposed to be used. Someone could test and implement fallbacks if they really want to use this ancient one. Either way this seems to be the most convenient way to display that `r19` is the first actually supported and working NDK.

> I guess we can remove the empty winit/glutin and flutter sections too.

They're gone now.

Reply:
> Someone could test and implement fallbacks if they really want to use this ancient one.

Nah, was just curious since you appeared to have gone through the trouble of actually testing it. I don't think anyone cares if r18 doesn't work. that's a release from 2018 right?

Reply:
Wasn't too hard to select a bunch of NDKs in the SDK manager and run through them with `ANDROID_NDK_ROOT=<the version> cargo apk run <some example>` :)

https://developer.android.com/ndk/downloads/revision_history September 2018 indeed!